### PR TITLE
virsh_snapshot_create_as: Fix syntax error in formatting string

### PR DESCRIPTION
**Pull request by KalaiIBM**
Found the wrong syntax in the code and fixed the same

```
TypeError: not enough arguments for format string
```

Signed-off-by: Kalaiarasan Panneerselvam <panneer1@in.ibm.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_disk.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def run(test, params, env):
             result = virsh.attach_disk(vm_name, source=img_path, target="vdf",
                                        extra=extra, debug=True)
             if result.exit_status:
-                test.cancel("Failed to attach disk %s to VM."
+                test.fail("Failed to attach disk %s to VM."
```
Comment (sathnaga) [inline libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_disk.py]: this file change is not related to commit, I guess mistaken, pls remove

Comment (KalaiIBM) [reply libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_disk.py]: fixed satheesh!
